index.html

```diff
@@ -7,6 +7,7 @@
 
   <body>
     <div id="app">
+      <h1>New Change</h1>
```

Comment:
looks like this is from my old demo code, can you update this branch with `main` branch so that this change doesn't show?